src/config_test.rs

```diff
@@ -180,25 +180,25 @@ mod get_table_key_value_from_config {
 
     #[test]
     fn handles_missing_table() {
-        let contents = "[hooks]";
+        let contents = r#"[hooks]"#;
```

Comment:
So we won't need to use raw strings in standard cases like this. The raw string lits are particularly helpful in cases where there's characters in the string that would have to be escaped (like toml with string values, json, etc.). We don't have anything here that needs escaping so we can leave it as a standard literal.
https://doc.rust-lang.org/stable/rust-by-example/std/str.html#literals-and-escapes 
```suggestion
        let contents = "[hooks]";
```